_posts/2020/2020-12-10-how-to-structure-css-classes.md

```diff
@@ -104,6 +111,10 @@ Another addition is to have states classes which are reflecting changes to a com
 ```
 
 #### SUIT CSS code example
+
+Do you remember the little author box we used as an example for BEM?
+This is how it would look like with SUIT CSS:
```

Comment:
`This is how it could look like with SUIT CSS:`
(I like "could" more and it is also not that dangerous for me if there would be a better way to write this component in SUIT... 😜)